### PR TITLE
add summarization script

### PR DESCRIPTION
**Pull request by yangyingxiang**

### CODE REVIEW

File: llm/summarize.py [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+url_get = "https://beta.yuanjian.org/api/v1/summaries.list"
+url_post = 'https://beta.yuanjian.org/api/v1/summaries.write'
+params = {"key": "原始文字", "excludeTranscriptsWithKey": "xxx_llm_1000"}
+headers_get = {"Authorization": "Bearer {}".format(os.environ['YUANJIAN_TOKEN'])}
+headers_post = ["Authorization: Bearer {}".format(os.environ['YUANJIAN_TOKEN']),
+                "Content-Type: application/x-www-form-urlencoded"]  #
+
+
+def get_list(url, headers, params):
+    """
```
Comment (weihanwang) [inline llm/summarize.py:41]: Can we format to two spaces per tab?

File: llm/summarize.py [hunk 2]
```diff
@@ -0,0 +1,233 @@
+"""
+Please set environment variable YUANJIAN_TOKEN for beta.yuanjian.org
```
Comment (weihanwang) [inline llm/summarize.py:2]: Change to INTEGRATION_AUTH_TOKEN to be consistent with the rest of  codebase.

File: llm/summarize.py (hunk 1 above)
Comment (yangyingxiang) [reply llm/summarize.py:41]: PEP 8 recommends using 4 spaces, can we keep using that in python code?

Comment (weihanwang) [reply llm/summarize.py:41]: sure thing. (was mentioning it because all js code in this repo use 2 spaces).

**Review by weihanwang — APPROVED** [review]

**Pull request merged**